### PR TITLE
Pic 3378

### PR DESCRIPTION
**Pull request by johnyfarrar**
Change the content on zero cases for outcomes workflow

### CODE REVIEW

File: server/views/outcomes/casesInProgress.njk [hunk 1]
```diff
@@ -138,7 +138,10 @@
       {{ govukTable(tableData) }}
     {% endblock %}
   {% else %}
-    <p class="govuk-body">{{ "There are no cases that match these criteria." if params.filtersApplied else "No cases in progress available." }}</p>
+  {% if params.filtersApplied %}
+      <span class="govuk-heading-m">0 cases</span>
```
Comment (rschallapalli) [inline server/views/outcomes/casesInProgress.njk:142]: Does the ticket mention this? I can see this on the ticket instead:

> 
>  If a user applies filters and there are no matching cases the content should read:
> 
> There are no matching cases.

File: server/views/outcomes/casesToResult.njk [hunk 2]
```diff
@@ -120,7 +120,7 @@
     {% if params.filtersApplied %}
       <span class="govuk-heading-m">0 cases</span>
```
Comment (rschallapalli) [inline server/views/outcomes/casesToResult.njk:121]: Same here, is this expected? captured somewhere else?

**Review by rschallapalli — CHANGES_REQUESTED** [review]

**Review by paddybasi — APPROVED** [review]
Aside from the comments @rschallapalli has mentioned it LTGM

File: server/views/outcomes/casesToResult.njk (hunk 2 above)
Comment (johnyfarrar) [reply server/views/outcomes/casesToResult.njk:121]: Ticket has been updated!

**Pull request merged**